### PR TITLE
Instructor view results: increase responses limit to 8000 #6586

### LINKED ISSUE #6586: Instructor view results: increase responses limit to 8000
Currently, a result page is not loaded if it involves more than 2000 responses. This limit is too low. Let's increase it to 8000.
The limit is defined in FeedbackSessionsLogic.java

### PR DESCRIPTION
**Pull request by gourmetbrian**
Fixes #6586

Changed 2000 to 8000 for response limit range.

Updated XML errors present in branch.

### CODE REVIEW

**Comment by Neurrone** [comment]
Hi @gourmetbrian, thanks for contributing to teammates. 

* In your PR description, you have 'Fixes # 6586'. Github only recognizes it without the space between the # and issue number, 'Fixes #6586'.
* PR title should be the issue's title followed by its number, 'Instructor view results: increase responses limit to 8000 #6586'.

Will fix these for you; please remember to do so in future :)

**Comment by Neurrone** [comment]
Fixes #6586

**Comment by junhaoyap** [comment]
@gourmetbrian it seems that the specific fix that is asked for is correct, there are also a lot of changes that should not have been made such as pmd changes and appstatsfilter, can you revert the unnecessary changes before we move forward with the status of the PR?

cc @wkurniawan07 for assigning relevant people for review

Also setting to ongoing

**Comment by gourmetbrian** [comment]
@junhaoyap I reverted the XML changes so the only change in this branch is now the original requirement. Sorry for the confusion on my part.

File: src/main/java/teammates/logic/core/FeedbackSessionsLogic.java [hunk 1]
```diff
@@ -856,8 +856,9 @@ public String getFeedbackSessionResultsSummaryInSectionAsCsv(
             String feedbackSessionName, String courseId, String userEmail,
             String section, String filterText, boolean isMissingResponsesShown, boolean isStatsShown)
             throws EntityDoesNotExistException, ExceedingRangeException {
+                      
+        long indicatedRange = section == null ? 8000 : -1;
```
Comment (Neurrone) [inline src/main/java/teammates/logic/core/FeedbackSessionsLogic.java]: Suggest adding a new constant value for this, something like INSTRUCTOR_RESPONSE_LIMIT?

**Comment by wkurniawan07** [comment]
Hi @gourmetbrian, any updates for this PR?

**Comment by gourmetbrian** [comment]
The XML changes I made have all been reverted. I introduced a constant for the 8000 figure.

**Comment by Neurrone** [comment]
@gourmetbrian changes to xml still show up in the files changed tab.

**Comment by gourmetbrian** [comment]
@Neurrone Those changes reflect my original alteration, but the xml fully matches what I got at the beginning. Do you recommend abandoning all changes via git revert to start anew?

**Comment by Neurrone** [comment]
@gourmetbrian yeah, that might be the fastest way, given that the actual fix should only be a few lines of changes.

**Comment by gourmetbrian** [comment]
@Neurrone Hi Tan--I'm worried with resetting all of these changes I will make another commit that has a huge number of diffs and will fail again. Just to confirm, I should commit to this branch instead of starting a fresh branch that only has the *one* change. Sorry for all the confusion I've caused.

**Comment by Neurrone** [comment]
@gourmetbrian yup, you can still use this branch. Try the following:

* Use git revert to roll back all changes
* Merge the latest changes from master, and check that you don't see any changes
* Apply your fix

**Comment by Neurrone** [comment]
Closing as its fixed in https://github.com/TEAMMATES/teammates/pull/6937

**Pull request closed without merging**